frontend/src/lib/dashbaordVariables/getDashboardVariables.ts

```diff
@@ -7,24 +7,27 @@ export const getDashboardVariables = (): Record<string, unknown> => {
 			globalTime,
 			dashboards: { dashboards },
 		} = store.getState();
-		const [selectedDashboard] = dashboards || [];
-		const {
-			data: { variables = {} },
-		} = selectedDashboard;
+		if (dashboards.length > 0) {
```

Comment:
Can `dashboards` ever be `null`/`undefined` here?

Reply:
no @srikanthccv initial value is always empty array ref: https://github.com/SigNoz/signoz/blob/develop/frontend/src/store/reducers/dashboard.ts#L26